src/lib/src/service/execution/test.execution.service.spec.ts

```diff
@@ -1,30 +1,44 @@
+import { ComponentFixture, TestBed, fakeAsync, tick } from '@angular/core/testing';
```

Comment:
uses fakeAsync, but never uses tick, so the former is probably unnecessary, as well.